android/lib/src/main/java/com/swmansion/gesturehandler/TapGestureHandler.java

```diff
@@ -4,15 +4,21 @@
 import android.view.MotionEvent;
 
 public class TapGestureHandler extends GestureHandler<TapGestureHandler> {
-
+  private static float MAX_VALUE_IGNORE = Float.MIN_VALUE;
   private static final long DEFAULT_MAX_DURATION_MS = 500;
   private static final long DEFAULT_MAX_DELAY_MS = 500;
   private static final int DEFAULT_NUMBER_OF_TAPS= 1;
 
+  private float mMaxDeltaX = MAX_VALUE_IGNORE;
+  private float mMaxDeltaY = MAX_VALUE_IGNORE;
+  private float mMaxDist = MAX_VALUE_IGNORE;
```

Comment:
In other handlers we store square of the dist (`mMaxDistSq`) would be nice to change it here for consistency.